### DISCUSSION TITLE
How much data do you share with OpenAI practically speaking?

### DISCUSSION THREAD

**Discussion by jamie9090** [brianpetro/obsidian-smart-connections#396] [General]
Amazing plugin.

Hope I can get some perspectives from people to shed some further light on OpenAI data processing for a layman from a practical perspective.

For an example, Open AI's TOS says that it won't use data for training, although it retains data for a maximum of 30 days. And that we retain ownership of inputs/outputs.

From a privacy perspective, how secure would you say the data is? For an example, do you personally share "everything" (within reason)? Journals? Business information etc

Just interested to hear others perspectives here.

**Comment by brianpetro** [comment]
Hi @jamie9090 

I've been working on V2 (should be released to supporters within days) and that defaults to using a local model for embedding.

Then, only notes (more accurately, segments of notes) used as context in the Smart Chat feature will be sent to OpenAI, which is a significant decrease compared to all notes being sent to the embedding API for processing.

I know that doesn't exactly answer your question, but I wanted to clarify the direction of the project since how it uses OpenAI APIs is relevant.

Furthermore, just in the past couple days, I've been seeing some reports of local chat models running on Mac M2 8GB machines (what I would consider relatively accessible consumer hardware). So, while I'm not focused on local chat models right now, this progress does look good for local chat models also being supported by Smart Connections sometime in 2024.

Thanks for your interest in the project!

🌴

**Reply by jamie9090** [reply]
Thanks! 

1. Is there a way to force clear OpenAI API after using SmartConnections?
2. Where can I learn more about the supporter option?

**Reply by brianpetro** [reply]
I'm not aware of any clearing mechanism. And the supporter details are in the settings for the plugin.

🌴
